crons.go

```diff
@@ -91,19 +91,20 @@ func runSentryCronsCheckin(ctx context.Context, job *batchv1.Job, eventHandlerTy
 		return errors.New("cannot find cronJob data")
 	}
 
-	// capture checkin event called for by informer handler
-	if eventHandlerType == EventHandlerAdd {
+	// The job just begun so check in to start
+	if job.Status.Active == 0 && job.Status.Succeeded == 0 && job.Status.Failed == 0 {
```

Comment:
Have you confirmed, does it actually mean that the corresponding pod has actually started? (versus just being scheduled)

Reply:
This only means that the job object has been created, but does not guarantee that a pod has actually started. I have some questions about this.

What should the granularity of a crons monitor be? Originally, I thought that a cronJob should checkin at the beginning and end of a job object (reporting the exit status). For example a cronJob may schedule a job that must run 3 pods to completion. Then we shouldn't check in three times, right? Since they are all associated with a single job instance.

Also, if we wanted to keep track of each time the program fails, wouldn't it be inconsistent based on the restart policy? If the restart policy is "Never" (as it is) then we do indeed track all program failures. However, if it is set on "OnFailure", then wouldn't just the container inside the pod restart? We still won't see the granularity of each container failure even if we track the pods.

Reply:
As discussed in Slack, let's focus on tracking job start/end only for now, and see how it works.